tableauserverclient/server/endpoint/metadata_endpoint.py

```diff
@@ -0,0 +1,32 @@
+from .endpoint import Endpoint, api
+from .exceptions import GraphQLError
+import logging
+import json
+
+logger = logging.getLogger('tableau.endpoint.metadata')
+
+
+class Metadata(Endpoint):
+    @property
+    def baseurl(self):
+        return "{0}/api/exp/metadata/graphql".format(self.parent_srv._server_address)
+
+    @api("3.2")
```

Comment:
@gaoang2148 what's the latest, is it 3.4?

Reply:
3.4 is 19.2, 3.5 is 19.3

Reply:
This also jumps out to me, minor issue so just merged!